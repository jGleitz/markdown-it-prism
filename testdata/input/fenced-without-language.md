# Test

This is a fenced code block:

```
public class Foo() {
	public Foo(bar) {
		System.out.println(bar);
	}
}
```
