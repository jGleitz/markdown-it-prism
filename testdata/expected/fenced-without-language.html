<h1>Test</h1>

<p>This is a fenced code block:</p>
<pre><code>
	public class Foo() {
		public Foo(bar) {
			System.out.println(bar);
		}
	}
</code></pre>
